src/common/config/constants.ts

```diff
@@ -307,3 +307,5 @@ export const OCP_STATIC_IP_DOC =
 export const AI_UI_TAG = 'ui_ocm';
 
 export const OCP_RELEASES_PAGE = 'openshift/releases';
+
+export const NUTANIX_CONFIG_LINK = 'https://access.redhat.com/solutions/6983944';
```

Comment:
Is this the private link you mentioned? 
Shouldn't this be a public link instead?

Reply:
It's the link that @vrutkovs provides to us. It's in the same place that vsphere link

Reply:
Solution is public